mtp_noms_ops/apps/security/tests/test_views.py

```diff
@@ -3035,6 +3033,53 @@ def test_get(self):
             self.assertContains(response, '123456******9876')
             self.assertContains(response, '02/20')
 
+    def test_check_view_hides_action_buttons_if_resolved_already(self):
+        """
+        There is currently nothing to prevent the view from showing a resolved security check.
+        Test that the accept/reject form is absent for resolved checks.
+        """
+        response_len = 0
+        check_id = 1
+        with responses.RequestsMock() as rsps:
+            self.login(rsps=rsps)
+            rsps.add(
+                rsps.GET,
+                api_url(f'/security/checks/{check_id}/'),
+                json=self.SENDER_CHECK_REJECTED
+            )
+            rsps.add(
+                rsps.GET,
+                urljoin(
+                    settings.API_URL,
+                    '/senders/{sender_profile_id}/credits/'.format(
```

Comment:
I've not used the Sentry `request-mock` library (i've tended to use this one instead https://pypi.org/project/requests-mock/), is it agnostic to query strings by default?

Reply:
it's a little odd. if you include a query string, it's required to match, if you don't then it doesn't matter. in this test, i just wanted to simplify it and not bother with query string matching as the intention of this test is to check a different thing

Reply:
That's good to know, otherwise i'm sure that would have tripped me up at some point

Reply:
My only thought is that it might confuse people that the query string is present here but not elsewhere. Might be worth adding https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/487#discussion_r446127320 (or something to that effect) as a code comment in the test maybe?